app/views/articles/index.html.erb

```diff
@@ -1,3 +1,8 @@
+<%= link_to_unless_current "English", locale: 'en' %>
+<%= link_to_unless_current "Svenska", locale: 'sv' %>
+
+<%= link_to t('login'), "/" %>
+
 <h1> The Viking Times </h1>
```

Comment:
Will it be called The viking times in all languages? otherwise add a translation for it

Reply:
Yes!